spec/haskell/src/SEL4/Object/Endpoint.lhs

```diff
@@ -47,10 +47,12 @@ This function performs an IPC send operation, given a pointer to the sending thr
 The normal (blocking) version of the send operation will remove a recipient from the endpoint's queue if one is available, or otherwise add the sender to the queue.
 
 > sendIPC blocking call badge canGrant canGrantReply canDonate thread epptr = do
+>         stateAssert invs'_asrt "assert that `invs'` holds"
```

Comment:
still not a fan of saying `stateAssert ... "assert that ... holds"` due to the redundancy, but I guess it fits with the other ones